### PR TITLE
Bloody Mary, hot sauce, tomato juice added

### PR DESCRIPTION
**Pull request by AelundNolen**
#### Summary

SUMMARY: Content "Added bloody mary, hot sauce, tomato juice"

#### Purpose of change

I thought such a basic cocktail as Bloody Mary should be in the game, it's really not much different from the Screwdriver we already have. My main aim with this is to get acquainted with contributing by doing something easy, so that I can go on to help out with something really useful next, hopefully. The whole thing was prompted by a conversation on the discord a couple days ago.

#### Describe the solution

I have added three items(bloody mary cocktail, tomato juice, hot sauce) and two recipes(bloody mary and tomato juice). I have also added tomato juice to all item groups which I could find which have other juices, as well as added hot sauce in the same manner to all item groups which had soy sauce.

The main piece of this pull request is the bloody mary cocktail itself: it requires either tomato juice(my invention), or V8, along with vodka and various choices of things in addition, like the hot sauce(traditionally used in most recipes for bloody mary IRL) or pepper, salt, option of salt or lemon(implying using the juice of the lemon).

The second item to be added is the hot sauce: my thinking behind creating it is to make it a spice, not edible in raw form(who would eat raw hot sauce?), much like chili powder, pepper, salt etc. 

The third item is the tomato juice: the recipe requires 5 tomatoes, as a rough guesstimate based on an online recipe I googled, and gives 6 portions of juice as return, as I've also made it require water. It requires sugar and salt, as that was again, the recipe I have found.

#### Describe alternatives you've considered

I could have made do with just adding the cocktail itself using existing stuff like V8 and existing spices, but I thought that tomato juice and hot sauce(especially the hot sauce) could be expanded upon in future and existing recipes, if anything as to add "flavour" to the game. I hope that is a satisfactory line of reasoning.

#### Testing

I have done all the testing I could think of, namely: verified that the new items and recipes work as intended, as well as verified that the items spawn in the game world. I could not think of any more tests for just three items and two recipes.

#### Additional context

It all really comes down to me wanting to contribute, but needing something simple to do first to prove to myself I can do it, as well as to try out the process of contribution.

### CODE REVIEW

File: data/json/items/comestibles/alcohol.json [hunk 1]
```diff
@@ -1396,5 +1396,33 @@
...
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 25,
+    "healthy": 1,
+    "addiction_potential": 4,
+    "calories": 70,
+    "description": "Tomato juice mixed with vodka and some spices.  Hangover cure of choice for many.",
+    "price": 600,
+    "price_postapoc": 250,
+    "material": [ "alcohol", "fruit" ],
```
Comment (LaVeyanFiend) [inline data/json/items/comestibles/alcohol.json:1419]:
```suggestion
    "material": [ "alcohol", "tomato" ],
```
Tomato has its own material.

File: data/json/items/comestibles/drink.json [hunk 2]
```diff
@@ -1174,5 +1174,29 @@
...
+    "container": "bottle_plastic",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 40,
+    "healthy": 2,
+    "calories": 36,
+    "description": "A thick red liquid, made from tomatoes.  Not to everyone's taste, it is a healthy and refreshing drink when chilled.",
+    "price": 90,
+    "price_postapoc": 40,
+    "material": [ "fruit" ],
```
Comment (LaVeyanFiend) [inline data/json/items/comestibles/drink.json:1194]:
```suggestion
    "material": [ "tomato" ],
```

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -7107,5 +7107,46 @@
...
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "result_mult": 6,
+    "time": "45 m",
+    "autolearn": true,
+    "batch_time_factors": [ 80, 4 ],
+    "qualities": [ { "id": "BOIL", "level": 1 } ],
+    "tools": [ [ [ "water_boiling_heat", 2, "LIST" ], [ "food_processor", 20 ] ] ],
+    "components": [
+      [ [ "tomato", 5 ] ],
```
Comment (Rail-Runner) [inline data/json/recipes/recipe_food.json:7146]: Using irradiated tomatoes should probably be an option here too.

Comment (431und) [reply data/json/recipes/recipe_food.json:7146]: Didn't realize there were irradiated tomatoes! Will change.

**Pull request merged**